src/components/MessageLogger/FaultsAndWarningList/FaultsAndWarningList.tsx

```diff
@@ -1,29 +1,45 @@
 import ConsoleList from "@components/MessageLogger/ConsoleList/ConsoleList";
+import { useState } from "react";
 import { RootState } from "store";
 import { useSelector } from "react-redux";
+import { store } from "../../../store";
 import styles from "@components/MessageLogger/FaultsAndWarningList/FaultsAndWarningList.module.scss";
-
+import { useInterval } from "@hooks/useInterval";
+import { Message } from "@models/Message";
 const warningColor = { h: 41, s: 100, l: 40 };
 const faultColor = { h: 0, s: 100, l: 40 };
 
 export const FaultsAndWarningList = () => {
-  const warningMessages = useSelector(
-    (state: RootState) => state.messages.warning
-  );
-  const faultMessages = useSelector((state: RootState) => state.messages.fault);
+    const [messages, setMessages] = useState({
+        fault: [] as Message[],
+        warning: [] as Message[],
+    });
+    useInterval(() => {
+        const state = store.getState();
+        setMessages({
+            fault: state.messages.fault,
+            warning: state.messages.warning,
+        });
+    }, 1000 / 70);
```

Comment:
Why 1000/ 70?

Reply:
This represents 70fps, but we have to measure what is the optimal time for the best performance